Milestone-4: Styling

Jmagero opened the pull request:
- [x] Add design match Zeplin's design.

Jmagero committed:
Add style to the root container

Jmagero committed:
Import style

Jmagero committed:
Add style to AddBook container

Jmagero committed:
Add style to display book container

Jmagero committed:
Add style to navBar

Jmagero committed:
Fix linter errors

MahmoudBakr23 commented on src/components/DisplayBooks.module.css:
```diff
@@ -0,0 +1,128 @@
+  margin: 0;
+}
+
+.current {
+  opacity: 0.5;
+  font-family: var(--font);
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.chapter {
+  font-family: var(--font);
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
```
- [ ] Kindly combine these two selectors because they're sharing almost the same rules, the DRY rule is about don't repeat yourself, and this is for cleaner and more optimized code 😄 🙏

_You can find about best practices right [here](https://speckyboy.com/good-bad-css-practices/)._

>Please apply these changes to all similar situations through your code files!

MahmoudBakr23 commented on src/components/NavBar.module.css:
```diff
@@ -0,0 +1,55 @@
+  text-decoration: none;
+}
+
+.BookstoreTitle {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: var(--icon-font);
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.BOOKS {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: var(--icon-font);
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  text-transform: uppercase;
+  text-decoration: none;
+}
```
- [ ] DRY rule right here 👍 
_Please see the comment above about the DRY rule._

MahmoudBakr23 commented on src/components/NavBar.module.css:
```diff
@@ -0,0 +1,55 @@
+  --font: robotoslab;
+}
+
+.NavBar {
+  display: flex;
+  justify-content: flex-start;
+  padding: 2px 5px;
+  align-items: center;
+  height: 3rem;
+  margin-bottom: 1rem;
+  background-color: #fff;
+  margin-top: 1rem;
+}
```
- [ ] Please adjust the styling of the Navbar to be separated from the main section in order to be exactly as the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). 😄 🙏 

![Screenshot (1)](https://user-images.githubusercontent.com/47625407/132089174-0b965112-0889-4390-b68d-9e4f888fb649.png)

MahmoudBakr23 commented on src/components/NavBar.module.css:
```diff
@@ -0,0 +1,55 @@
+.root {
+  --icon-font: montserrat;
+  --font: robotoslab;
```
- [ ] Kindly use the same fonts as shown in the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). 😄 🙏 

>Please apply these changes to all of your elements accordingly!

MahmoudBakr23 commented on src/components/DisplayBooks.js:
```diff
@@ -12,23 +14,34 @@ const DisplayBook = (props) => {
+              <p className={styles.title}>
+                { book.title }
+              </p>
+              <p className={styles.category}>
```
- [ ] The book category should be styled as shown in the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). 😄 🙏 

![Screenshot (1)](https://user-images.githubusercontent.com/47625407/132089241-86921327-b144-42bd-8663-4501af7ac780.png)

MahmoudBakr23 commented on src/components/NavBar.js:
```diff
@@ -17,14 +18,14 @@ const NavBar = () => {
     <>
-      <nav>
-        <h1>
+      <nav className={styles.NavBar}>
```
- [ ] Please add the icon at the top-right corner of your navbar as shown in the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). 😄 🙏

MahmoudBakr23 reviewed (changes requested):
Hello @Jmagero 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct
- Descriptive README, PR headline, summary, and commit messages
- Extra credit for making the functionalities of the app in this milestone

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] It'd be much nicer if you attached an image of your app to the README file for a completely professional and good-looking repo. 😉 

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Jmagero committed:
Install react-icons

Jmagero committed:
Fix book category style & applied DRY

Jmagero committed:
Style Nav to original design

Jmagero committed:
Fix linter errors

Jmagero committed:
Add image

Jmagero commented:
- [x]  Added an image of my app to the README file
- [x] Adjusted the styling of the Navbar to be separated from the main section
- [x]  Used the same fonts as shown in the original design.
- [x] Added the icon at the top-right corner of your navbar as shown in the original design

ZawadiSumba66 reviewed (approved):
Hi @Jmagero 👋 , 
## STATUS: APPROVED 👍 
Great job implementing the changes requested by the previous reviewer. 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
